CMakeLists.txt

```diff
@@ -224,4 +224,38 @@ if (MSVC)
   set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "/incremental:no")
   set(CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO "/incremental:no /debug")
   set(CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL "/incremental:no")
+
+  # Older MSVC versions don't support building in parallel.
+  # Force non-parallel builds of isle and lego1 by putting them in a pool with 1 available job.
+  if(CMAKE_CXX_COMPILER_ID VERSION_LESS 12)
+    set_propertY(GLOBAL PROPERTY JOB_POOLS "msvc_lego=1;msvc_lego1=1")
```

Comment:
Minor typo (capitalized `Y`)

Reply:
The capital is for correct pronunciation :sweat_smile:

Reply:
This is fixed now.